tests/ecosystem/deployment/test_deployment.py

```diff
@@ -39,3 +46,38 @@ def test_deployment(pvc_factory, pod_factory):
         log.info(
             "Cluster will be destroyed during teardown part of this test."
         )
+
+
+@skipif_no_encrypted
+@deployment
+def test_verify_osd_encryption():
+    """
+    Verify the OSD is encrypted
+    """
+    ocs_version = float(config.ENV_DATA['ocs_version'])
+    if ocs_version < 4.6:
+        error_message = "Encryption at REST can be enabled only on OCS >= 4.6!"
+        log.error(error_message)
```

Comment:
Remove this log statement, you're already including this message in the raised error.